### ISSUE TITLE
[header] go.home link broken

### ISSUE THREAD

**Issue by landryb** [georchestra/georchestra#1300]
When on a georchestra webapp on sdi.georchestra.org, clicking the georchestra logo on the top left leads to a blank page with only the header shown: url bar shows https://sdi.georchestra.org/header/?lang=en&active=mapfishapp# and it seems only the header is loaded, no iframe.

**Comment by fvanderbiest** [comment]
Instead it should link to https://sdi.georchestra.org/

The culprit is https://github.com/georchestra/georchestra/blob/15.12/header/src/main/filtered-resources/WEB-INF/jsp/index.jsp#L272 which should use `shared.homepage.url` instead of `#`

**Comment by fvanderbiest** [comment]
Fixed on master for 16.12:
```html
        <a href="/" id="go_home" title="<fmt:message key='go.home'/>">
            <img src="img/logo.png" alt="<fmt:message key='logo'/>" height="50"/>
        </a>
```
